digital-twins/README.md

```diff
@@ -0,0 +1,9 @@
+# Azure IoT Plug-and-Play Preview
```

Comment:
Branding guidelines are "IoT Plug and Play" - note no "Azure" - here and elsewhere.